games/paclear/Portfile

```
@@ -0,0 +1,113 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/orangekame3/paclear 0.0.12 v
```

This looks good. Only nitpick would be to add a `revision` field:

```suggestion
go.setup            github.com/orangekame3/paclear 0.0.12 v
revision            0
```